test/pseudo-tty/repl-dumb-tty.js

```diff
@@ -1,36 +1,42 @@
 'use strict';
 const common = require('../common');
+const process = require('process');
 
-process.env.TERM = 'dumb';
+// TODO (fix): So this work on darwin/linux?
+if (process.env === 'windows') {
+  process.env.TERM = 'dumb';
```

Comment:
@ronag I think we want this test to work, and this change here already invalidates – the whole *point* is to test the REPL with `TERM=dumb`.

Reply:
Unfortunately I can't fix this test.

Reply:
It's going to block progress here indefinitely as it looks now.

Reply:
I can take a look at it. But as mentioned above, I still think it’s the streams timing that makes the difference here.

Reply:
Unfortunately I don't know what do with it even if I know it's a streams timing issue. Sorry and thank you.

Reply:
@ronag Can you not just update the expected test output to the current actual output? i.e.

```js
diff --git a/test/pseudo-tty/repl-dumb-tty.out b/test/pseudo-tty/repl-dumb-tty.out
index 3304faff0a4f..2bf4ff21a54b 100644
--- a/test/pseudo-tty/repl-dumb-tty.out
+++ b/test/pseudo-tty/repl-dumb-tty.out
@@ -1,4 +1,5 @@
-> console.log("foo")
+> > 
+console.log("foo")
 foo
 undefined
 > 1 + 2
@@ -10,6 +11,5 @@ undefined
 undefined
 > { a: 1 }
 { a: 1 }
->
+> 
 > .exit
->
```

it makes the test pass for me and should be within the range of what's allowed here as a breaking change

Reply:
Applied. Let's see what CI says.